CTM-ETM coupling 5b deliverable 2 & 3

kndehaan opened the pull request:
This PR contains updates of user output for deliverable 2&3 of the CTM-ETM coupling projects. The focus is to update outputs considering changes in modelling of:

- methanol
- waste mix
- oil mix
- ammonia

The (queries for the) following elements are updated:

- Data-exports
- Charts
- Dashboard items
- MYC

Goes with:
- https://github.com/quintel/etsource/pull/3093
- https://github.com/quintel/etmodel/pull/4302
- https://github.com/quintel/multi-year-charts/pull/49

mabijkerk commented:
As far as I can tell, in the multi-year-charts, all final demand queries should represent total demand (energetic & non-energetic). However, I am seeing some queries for which this is not the case. For example, `myc_final_demand_of_liquid_biofuels_in_industry` explicitly filters on energetic demand. This is an issue, since we have just added non energetic final demand of e.g., biokerosene. @kndehaan can you look into this?

mabijkerk commented on data/charts/renewables.ts:
```diff
@@ -14,7 +14,8 @@ export default {
         'myc_final_demand_of_renewable_liquid_biofuels',
-        'myc_final_demand_of_renewable_solid_biofuels'
+        'myc_final_demand_of_renewable_solid_biofuels',
+        'myc_final_demand_of_renewable_waste'
```
Should renewable methanol be added here? Just like synthetic kerosene it can be made from renewable electricity and hydrogen.

kndehaan commented:
> However, I am seeing some queries for which this is not the case. For example, myc_final_demand_of_liquid_biofuels_in_industry explicitly filters on energetic demand. This is an issue, since we have just added non energetic final demand of e.g., biokerosene.

Well spotted! I added some new general final demand mece queries for multiple energy carriers per sector, where no filtering of energetic is applied (so both final demand of energetic and non-energetic are considered). I updated the MYC queries where needed referring to these general queries.
